user-guide/singularity.rst

```diff
@@ -169,31 +169,32 @@ The process for using an image interactively on the compute nodes is very simila
 using them on the login nodes. The only difference is that you have to submit an interactive
 serial job to get interactive access to the compute node first.
 
-For example, to reserve a full node for you to work on interactively you would use:
+For example, to reserve a full node for you to work on interactively you would use something like:
 
 ::
 
-   [user@cirrus-login0 ~]$ qsub -IVl select=1:ncpus=36,walltime=0:20:0,place=scatter:excl -A t01
-   qsub: waiting for job 234192.indy2-login0 to start
-
+   [user@cirrus-login0 ~]$ salloc --exclusive --nodes=1 --tasks-per-node=36 --cpus-per-task=1 --time=00:20:00 --account=[budget code] 
```

Comment:
Do you not need a partition and qos here too?

Reply:
I took that salloc command from here, https://cirrus.readthedocs.io/en/master/user-guide/batch.html#interactive-jobs-salloc . Does that page need updating too?